handlers/handler-bizkeeper/src/main/java/org/apache/servicecomb/bizkeeper/ThrowExceptionFallbackPolicy.java

```diff
@@ -30,7 +30,7 @@ public String name() {
   }
 
   @Override
-  public Response getFallbackResponse(Invocation invocation) {
+  public Response getFallbackResponse(Invocation invocation, Throwable error) {
```

Comment:
need change second parameter from null to error of createBizkeeperException, or not?

Reply:
It's better to use error as the parameter, I think